starport-blog/01-index.md

```diff
@@ -0,0 +1,345 @@
+# Create posts
+
+By following this beginner tutorial, you will end up with a simple blog app that is powered by the Cosmos SDK.
+
+## Prerequisites
+
+- A [recent version of Go](https://golang.org/doc/install) installed.
+- [Node.js](https://nodejs.org/) installed.
+
+## Getting Started
+
+Let's get started! The first step is to [install the `starport](https://github.com/tendermint/starport) CLI tool.
+
+After `starport` is installed, use it to create the initial app structure inside a directory named `blog`:
+
+```
+starport app github.com/example/blog
+```
+
+One of the main features of Starport is code generation. The command above has generated a directory structure with a working blockchain application. Starport can also add data types to your app with `starport type` command. To see it in action, follow the poll application tutorial. In this guide, however, we'll create those files manually to understand how it all works under the hood.
+
+## Overview
+
+Let's take a quick look at what Starport has generated for us. `app/app.go` file imports and configures SDK modules and creates a constructor for our application that extends a [basic SDK application](https://docs.cosmos.network/master/core/baseapp.html) among other things. This app will use only a couple standard modules bundled with Cosmos SDK (including `auth` for dealing with accounts and `bank` for handling coin transfers) and one module (`x/blog`) that will contain custom functionality.
+
+In `cmd` directory we have source files of two programs for interacting with our application: `blogd` can configure genesis and launch the app and `blogcli` contains commands to send transactions and start an HTTP API server.
```

Comment:
`blogd` starts a full-node for your blockchian

`blogcli`  enables you to query the full-node, either to update the state by sending a transaction or to read it via a query